analyses/snv-callers/README.md

```diff
@@ -0,0 +1,211 @@
+Mutations are assigned all annotations that they overlap (using  
+  `GenomicRanges::overlap`).
+
+*Output for this analysis*
+* `results/<caller_name>/<caller_name>_regions.tsv`
+* `plots/<caller_name>/<caller_name>_<strategy>_snv_regions.png`
+
+### Tumor Mutation Burden Calculation
+
+To calculate TMB, the sum of the bases included in the WXS or WGS BED regions
+are used as the denominator, depending on the sample's processing strategy.
+
+```
+TMBwxs = sum(mutation_w-in_bedwxs)/(wxs_genome_size/1000000)
+TMBwgs = sum(all_mutations)/(wgs_genome_size/1000000)
+```
+
+Where genome size is calculated from the respective BED file as:
+```
+genome_size = sum(End_Position - Start_Position)
+```
+
+BED regions for WXS samples can be [found here](https://raw.githubusercontent.com/AstraZeneca-NGS/reference_data/master/hg38/bed/Exome-AZ_V2.bed).
+BED regions used for WGS samples are caller specific are from <unknown as of now>
+The sample-wise TMB calculations written to a TSV ending in `_tmb.tsv` in the
+caller's results folder.
+
+*Output for this analysis*
+* `results/<caller_name>/<caller_name>_tmb.tsv`
+* `plots/<caller_name>/<caller_name>_<strategy>_tmb_plot.png`
+
+### COSMIC Mutation Overlap
+
+The COSMIC mutation data were obtained from https://cancer.sanger.ac.uk/cosmic/download
+*To run this analysis, you need to obtain these data.*
+The full, unfiltered somatic mutations file `CosmicMutantExport.tsv` for grch38
+is used here and the genomic coordinates is arranged to be in BED format.
+COSMIC mutations are overlapped with the present data's mutations using
+GenomicRanges. The outcome of this overlap is added to the VAF data.frame with
+two TRUE/FALS columns: `overlap_w_cosmic` is TRUE for mutations that overlap
+with COSMIC mutations, while `same_as_cosmic` is TRUE when the base change
+summary is also identical. The VAF for mutations that are or are not overlapping
+with COSMIC mutations are then plotted in a violin plot.
+
+*Output for this analysis*
+* `results/<caller_name>/<caller_name>_vaf.tsv`
+* `plots/<caller_name>/<caller_name>_<strategy>_cosmic_plot.png`
+
+## Comparison of Callers - coming soon
+
+### Mutation IDs  
+
+In order to compare mutations across callers, I created a `mutation_id` from
+combining information from standard MAF fields.
+
+`mutation_id` is a concatenation of:  
+* `Hugo_Symbol`  
+* [`change`](#base-change-analysis)  
+* `Start_Position`  
+* `Tumor_Sample_Barcode` (the sample ID)  
+
+If mutation_id's are identical among MAF files, they are considered the same.
+
+## Overall file structure
+```
+OpenPBTA-analysis
+├── analyses
+│   └── snv-callers
+│       ├── run_variant_caller_reports.R
+│       ├── scripts
+│       │   ├── 00-set-up.R
+│       │   ├── 01-calculate_vaf_tmb.R
+│       │   └── 02-run_eval.R
+│       ├── functions
```

Comment:
```suggestion
│       ├── util
```
Is what we have over on the dimension reduction stuff

Reply:
Funny enough I had it named that originally and then changed it. lol.